libnavigation-core/src/main/java/com/mapbox/navigation/core/OnboardRouterFiles.kt

```diff
@@ -33,14 +33,11 @@ internal class OnboardRouterFiles(
         }
     }
 
-    private fun defaultFilePath(options: OnboardRouterOptions): String {
-        val tilesUri = options.tilesUri
-        val tilesVersion = options.tilesVersion
-        val directoryVersion = "Offline/${tilesUri.host}/$tilesVersion/tiles"
```

Comment:
Why did we change the directory name structure?

Reply:
We shall now have `Offline/${tilesUri.host}/$tilesVersion`, where you specify only the `Offline/` - the rest will be appended by NN. This is also a good transition for 2.x where a similar approach is already being used.

Reply:
Makes sense. We should specify in the docs then that these are created internally.

Reply:
Could we add a quick note on this in the docs?

Reply:
discussed: we don't control it, so it no sense to leave a comment